include/plansys2_tfd_plan_solver/tfd_plan_solver.hpp

```diff
@@ -29,14 +29,20 @@ class TFDPlanSolver : public PlanSolverBase
 public:
   TFDPlanSolver();
 
-  void configure(rclcpp_lifecycle::LifecycleNode::SharedPtr & node, const std::string & id);
+  void configure(rclcpp_lifecycle::LifecycleNode::SharedPtr, const std::string &);
```

Comment:
Found a bug where this extra `&` was actually a different function signature than what `PlannerNode` uses, so none of the code in this `configure()` function was actually running -- oops :)